e-text-field-type.yml

```
@@ -1,1856 +1,1857 @@
+      - "DL_CLASS_CODE_S_TO"
+      - "DL_CLASS_CODE_TB_FROM"
+      - "DL_CLASS_CODE_TB_NOTES"
+      - "DL_CLASS_CODE_TB_TO"
+      - "DL_CLASS_CODE_TM_FROM"
+      - "DL_CLASS_CODE_TM_NOTES"
+      - "DL_CLASS_CODE_TM_TO"
+      - "DL_CLASS_CODE_TR_FROM"
+      - "DL_CLASS_CODE_TR_NOTES"
+      - "DL_CLASS_CODE_TR_TO"
+      - "DL_CLASS_CODE_TV_FROM"
+      - "DL_CLASS_CODE_TV_NOTES"
+      - "DL_CLASS_CODE_TV_TO"
+      - "DL_CLASS_CODE_V_FROM"
+      - "DL_CLASS_CODE_V_NOTES"
+      - "DL_CLASS_CODE_V_TO"
+      - "DL_CLASS_CODE_W_FROM"
+      - "DL_CLASS_CODE_W_NOTES"
+      - "DL_CLASS_CODE_W_TO"
+      - "URL"
+      - "CALIBER"
+      - "MODEL"
+      - "MAKE"
+      - "NUMBER_OF_CYLINDERS"
+      - "SURNAME_OF_HUSBAND_AFTER_REGISTRATION"
+      - "SURNAME_OF_WIFE_AFTER_REGISTRATION"
+      - "DATE_OF_BIRTH_OF_WIFE"
+      - "DATE_OF_BIRTH_OF_HUSBAND"
+      - "CITIZENSHIP_OF_FIRST_PERSON"
+      - "CITIZENSHIP_OF_SECOND_PERSON"
+      - "CVV"
+      - "DATE_OF_INSURANCE_EXPIRY"
+      - "MORTGAGE_BY"
+      - "OLD_DOCUMENT_NUMBER"
+      - "OLD_DATE_OF_ISSUE"
+      - "OLD_PLACE_OF_ISSUE"
+      - "DL_CLASS_CODE_LR_FROM"
+      - "DL_CLASS_CODE_LR_TO"
+      - "DL_CLASS_CODE_LR_NOTES"
+      - "DL_CLASS_CODE_MR_FROM"
+      - "DL_CLASS_CODE_MR_TO"
+      - "DL_CLASS_CODE_MR_NOTES"
+      - "DL_CLASS_CODE_HR_FROM"
+      - "DL_CLASS_CODE_HR_TO"
+      - "DL_CLASS_CODE_HR_NOTES"
+      - "DL_CLASS_CODE_HC_FROM"
+      - "DL_CLASS_CODE_HC_TO"
+      - "DL_CLASS_CODE_HC_NOTES"
+      - "DL_CLASS_CODE_MC_FROM"
+      - "DL_CLASS_CODE_MC_TO"
+      - "DL_CLASS_CODE_MC_NOTES"
+      - "DL_CLASS_CODE_RE_FROM"
+      - "DL_CLASS_CODE_RE_TO"
+      - "DL_CLASS_CODE_RE_NOTES"
+      - "DL_CLASS_CODE_R_FROM"
+      - "DL_CLASS_CODE_R_TO"
+      - "DL_CLASS_CODE_R_NOTES"
+      - "DL_CLASS_CODE_CA_FROM"
+      - "DL_CLASS_CODE_CA_TO"
+      - "DL_CLASS_CODE_CA_NOTES"
+      - "CITIZENSHIP_STATUS"
+      - "MILITARY_SERVICE_FROM"
+      - "MILITARY_SERVICE_TO"
+      - "DL_CLASS_CODE_NT_FROM"
+      - "DL_CLASS_CODE_NT_TO"
+      - "DL_CLASS_CODE_NT_NOTES"
+      - "DL_CLASS_CODE_TN_FROM"
+      - "DL_CLASS_CODE_TN_TO"
+      - "DL_CLASS_CODE_TN_NOTES"
+      - "DL_CLASS_CODE_D3_FROM"
+      - "DL_CLASS_CODE_D3_TO"
+      - "DL_CLASS_CODE_D3_NOTES"
+      - "ALT_DATE_OF_EXPIRE"
+      - "DL_CLASS_CODE_CD_FROM"
+      - "DL_CLASS_CODE_CD_TO"
+      - "DL_CLASS_CODE_CD_NOTES"
+      - "ISSUER_IDENTIFICATION_NUMBER"
+      - "PAYMENT_PERIOD_FROM"
+      - "PAYMENT_PERIOD_TO"
+      - "VACCINATION_CERTIFICATE_IDENTIFIER"
+      - "FIRST_NAME"
+      - "DATE_OF_ARRIVAL"
+      - "FT_SECOND_NAME"
+      - "FT_THIRD_NAME"
+      - "FT_FOURTH_NAME"
+      - "FT_LAST_NAME"
+      - "RM_FROM"
+      - "RM_NOTES"
+      - "RM_TO"
+      - "PW_FROM"
+      - "PW_NOTES"
+      - "PW_TO"
+      - "EB_FROM"
+      - "EB_NOTES"
+      - "EB_TO"
+      - "EC_FROM"
+      - "EC_NOTES"
+      - "EC_TO"
+      - "EC1_FROM"
+      - "EC1_NOTES"
+      - "EC1_TO"
```

Harmonise with other DL class codes
```suggestion
      - "DL_CLASS_CODE_RM_FROM"
      - "DL_CLASS_CODE_RM_NOTES"
      - "DL_CLASS_CODE_RM_TO"
      - "DL_CLASS_CODE_PW_FROM"
      - "DL_CLASS_CODE_PW_NOTES"
      - "DL_CLASS_CODE_PW_TO"
      - "DL_CLASS_CODE_EB_FROM"
      - "DL_CLASS_CODE_EB_NOTES"
      - "DL_CLASS_CODE_EB_TO"
      - "DL_CLASS_CODE_EC_FROM"
      - "DL_CLASS_CODE_EC_NOTES"
      - "DL_CLASS_CODE_EC_TO"
      - "DL_CLASS_CODE_EC1_FROM"
      - "DL_CLASS_CODE_EC1_NOTES"
      - "DL_CLASS_CODE_EC1_TO"
```